src/js/components/leftPanel/dataPanel/DataTabFooter.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+const DataTabFooter = (): JSX.Element => {
+  return (
+    <div className="data-tabview-footer">
+      <button onClick={(): void => console.log('Print Report!')}>
+        Print Report{' '}
```

Comment:
Idk if we still need the `{' '}` now that our `.data-tabview-container .data-tabview-footer button img` has the proper `margin-left`